packages/next/src/build/next-config-ts/utils.ts

```diff
@@ -0,0 +1,71 @@
+import type { Options as SWCOptions } from '@swc/core'
+import type { CompilerOptions } from 'typescript'
+
+import { resolve } from 'node:path'
+import { parseJsonFile } from '../load-jsconfig'
+
+export function resolveSWCOptions(cwd: string): SWCOptions {
```

Comment:
feel better to have a different name related to next.coinfig as there's already a helper with same name exported. should avoid the potential conflicts and would make reading code easier